charts/internal/shoot-storageclasses/templates/storageclasses.yaml

```diff
@@ -18,4 +18,13 @@ metadata:
 allowVolumeExpansion: true
 provisioner: ebs.csi.aws.com
 volumeBindingMode: WaitForFirstConsumer
+---
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass
+metadata:
+  name: gardener-default
+  annotations:
+    snapshot.storage.kubernetes.io/is-default-class: “true”
```

Comment:
Should we also add the `resources.gardener.cloud/delete-on-invalid-update=true` annotation? (ref https://github.com/gardener/gardener-resource-manager/pull/69)
/cc @rfranzke

Reply:
The `VolumeSnapshotClass` is a CRD, but we don't install a validating webhook for it. Maybe one of the CSI snapshot components creates it on its own.
But yes, we can simply add the annotation as well due to the similarly with normal storage classes.

Reply:
Ah. I just thought about the similarity to the storage classes and thought that `VolumeSnapshotClass`es might also be immutable. But I missed, that this is a CR...
Well, then we don't have to add it, I guess...